mobility/mobility/CMakeLists.txt

```diff
@@ -54,28 +48,39 @@ include_directories(
 )
 
 ## Declare a C++ executable: plangen
-add_executable(plangen tools/plangen.cc)
-add_dependencies(plangen ${catkin_EXPORTED_TARGETS})
-target_link_libraries(plangen
-  gflags glog ${catkin_LIBRARIES})
+# add_executable(plangen tools/plangen.cc)
+# add_dependencies(plangen ${catkin_EXPORTED_TARGETS})
+# target_link_libraries(plangen
+#   gflags glog ${catkin_LIBRARIES})
 
 ## Declare a C++ executable: teleop
 add_executable(teleop tools/teleop.cc)
-add_dependencies(teleop ${catkin_EXPORTED_TARGETS})
+ament_target_dependencies(teleop rclcpp ff_common ff_util ff_msgs jsonloader)
```

Comment:
From the list of packages we found earlier, I think we forgot to list a bunch as dependencies?

Reply:
added only used dependencies